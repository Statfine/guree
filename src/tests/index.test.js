describe("打包组件", () => {
  it("正确导出组件包", () => {
    const guree = require("..");
    expect(Object.keys(guree)).toMatchSnapshot();
  });
});
