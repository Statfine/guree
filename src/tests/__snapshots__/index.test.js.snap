// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`打包组件 正确导出组件包 1`] = `
Array [
  "Button",
  "Icon",
  "Row",
  "Col",
  "Modal",
  "Input",
  "Radio",
  "Checkbox",
  "Switch",
]
`;
